api/src/main/java/org/apache/cloudstack/api/command/user/config/ListCapabilitiesCmd.java

```diff
@@ -69,6 +69,8 @@ public void execute() {
         response.setInstancesStatsUserOnly((Boolean) capabilities.get(ApiConstants.INSTANCES_STATS_USER_ONLY));
         response.setInstancesDisksStatsRetentionEnabled((Boolean) capabilities.get(ApiConstants.INSTANCES_DISKS_STATS_RETENTION_ENABLED));
         response.setInstancesDisksStatsRetentionTime((Integer) capabilities.get(ApiConstants.INSTANCES_DISKS_STATS_RETENTION_TIME));
+        response.setStorageFsVmMinCpuCount((Integer)capabilities.get("storageFsVmMinCpuCount"));
```

Comment:
Can we use same constants that are used in ApiCmd classes?

Reply:
yes